tests/bin/setup_db.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
```

Comment:
The new installation process for unittests does this. This file is not necessary. Please delete.